client/src/Components/Pages/Questionnaire/PersonalDetails2/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+
+import './style.css'
+import '../style.css'
+
+const options = [
+
+    { value: '1', label: '1' },
+    { value: '2', label: '2' },
+    { value: '3', label: '3' },
+    { value: '4', label: '4' },
+    { value: '5', label: '5' },
+ 
+  ];
+
+class PersonalDetails2 extends Component {
+
+    state = {
+        selectedOption: null, 
+    }
+
+    handleChange = (selectedOption) => {
+
+        this.setState();
+        
+      }
+
+    Selector = () => {
+       return <Select
+        onChange={this.handleChange}
+        value={this.selectedOption}
+        options={options}
+    />
+    }
+      
+    render() {
+        return ( 
+            <div className="container">
+                <div className="fourth-section">
+                    <h2>Ok, so that's your school studies and hobbies done; now what's important in life to you ?</h2>
+                    <div className="eight-card">
+                        <p>What do you think is most important to you in a career? Tick your top 3</p>
+                        <div>
+                            <label>
+                                <input type="checkbox"/>Making money 
+                            </label>
+                            <Selector /> 
```

Comment:
I saw that you have a `Selector` class at the bottom of this file and also a `Selector` function inside the PersonalDetails2 class.  Both approaches will work - if you wanted to use the function instead of the class, you'd change this line to:
```{ this.Selector() }```

My general rule is that, if some piece of UI is used by multiple components / files, put it into its own class, but if it's only used in one component / file, just put it into a function inside that file.  But like I said, either way works.

Reply:
Great rule, I will try to implement it.